optimum/quanto/tensor/qbytes_ops.py

```diff
@@ -64,15 +64,15 @@ def _to_copy(op, t, dtype=None, **kwargs):
     out_data = op(t._data, dtype=t._data.dtype, **kwargs)
     # Apply the new dtype on the scale only
     out_scale = op(t._scale, dtype=dtype, **kwargs)
-    return QBytesTensor(t.qtype, t.axis, t.size(), t.stride(), out_data, out_scale)
+    return QBytesTensor.create(t.qtype, t.axis, t.size(), t.stride(), out_data, out_scale)
 
 
 @register_qbytestensor_op([torch.ops.aten.detach])
 def detach(op, t):
     # Detach both data and scale
     out_data = op(t._data)
     out_scale = op(t._scale)
-    return QBytesTensor(t.qtype, t.axis, t.size(), t.stride(), out_data, out_scale)
+    return QBytesTensor.create(t.qtype, t.axis, t.size(), t.stride(), out_data, out_scale)
```

Comment:
Not sure how bad it is to add `create` here everywhere? Once frozen, would a model still go in these aten extensions?